src/store/filestore.rs

```diff
@@ -197,21 +196,22 @@ impl<S: Store> ClientFilestore<S> {
             })
             .ok())
     }
-    fn read_dir_next_impl<F: LfsStorage + 'static>(
+    fn read_dir_next_impl(
         &mut self,
         state: ReadDirState,
-        fs: &'static Fs<F>,
     ) -> Result<Option<(DirEntry, ReadDirState)>> {
+        // TODO: check what happens (or used to happen) for FS != IFS
```

Comment:
Why?

Reply:
I think I saw a place where this was always called with the IFS even for other locations, but I can’t find it any more.  Maybe I accidentally looked at the `locate_file` implementation.

Reply:
Removed the comment.